scripts/dist.ts

```diff
@@ -230,6 +233,15 @@ function toDist(sourcePath: string): YAML.Document {
     }
   }
 
+  if (computeFromWasExplicitlySet) {
+    if (groups.size === 1) {
+      logger.error(
+        `${id}: uses compute_from which must not be used when the overall status does not differ from the per-key statuses. Delete this override.`,
```

Comment:
This works for me. I proposed the compute_from to the last sentence to make it more clear what needs to happen here but I don't feel strongly about it.

```suggestion
        `${id}: uses compute_from which must not be used when the overall status does not differ from the per-key statuses. Delete the compute_from override.`,
```

Reply:
Ah, good idea. I've opened https://github.com/web-platform-dx/web-features/pull/1618 to do this across all the dist errors.